### PR TITLE
Update Chinese translation

### PR DESCRIPTION
**Pull request by shincurry**
translation for zh portal and search

### CODE REVIEW

File: ghost/i18n/locales/zh/portal.json [hunk 1]
```diff
@@ -132,22 +132,22 @@
...
+    "This site is not accepting payments at the moment.": "本网站目前暂不接受付款。",
     "To complete signup, click the confirmation link in your inbox. If it doesn't arrive within 3 minutes, check your spam folder!": "要完成注册，请点击您收件箱中的确认链接。如果在3分钟内没有收到，请检查一下您的垃圾邮件文件夹！",
-    "To continue to stay up to date, subscribe to {{publication}} below.": "",
+    "To continue to stay up to date, subscribe to {{publication}} below.": "如需持续获取最新资讯，请在下方订阅{{publication}}",
     "Try free for {{amount}} days, then {{originalPrice}}.": "{{amount}}天免费试用，之后{{originalPrice}}。",
     "Unlock access to all newsletters by becoming a paid subscriber.": "成为付费订阅用户以解锁全部快报。",
     "Unsubscribe from all emails": "取消所有邮件订阅",
     "Unsubscribed": "取消订阅",
-    "Unsubscribed from all emails.": "",
+    "Unsubscribed from all emails.": "取消所有邮件订阅",
```
Comment (cathysarisky) [inline ghost/i18n/locales/zh/portal.json:150]: Could you please recheck this one - it should describe the status of a user (the user is unsubscribeD), not offer the user an action to take.  (The action is above: "Unsubscribe from all emails")

**Review by cathysarisky — APPROVED** [review]
Reviewed with AI help - one question to resolve, but then it looks merge-ready! :)

File: ghost/i18n/locales/zh/portal.json (hunk 1 above)
Comment (shincurry) [reply ghost/i18n/locales/zh/portal.json]: misled by the `Unsubscribed` translation, both fixed.

**Review by cathysarisky — APPROVED** [review]
No further concerns, and thank you for your work!

**Pull request merged**